ios/brave-ios/Sources/Brave/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -89,6 +125,11 @@ public class SearchViewController: SiteTableViewController, LoaderListener {
   )
 
   weak var searchDelegate: SearchViewControllerDelegate?
+  var profile: Profile! {
```

Comment:
How come it's `Profile!` and not just `Profile` or `Profile?`

Reply:
not sure this is added. will remove it since it is not needed in this class